plugin/PlayAssetDelivery/src/test/java/com/google/play/core/godot/assetpacks/utils/PlayAssetDeliveryUtilsTest.java

```diff
@@ -0,0 +1,63 @@
+package com.google.play.core.godot.assetpacks.utils;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import com.google.android.play.core.assetpacks.AssetPackState;
+
+import org.godotengine.godot.Dictionary;
+import org.junit.Test;
+
+public class PlayAssetDeliveryUtilsTest {
+  private Dictionary constructAssetPackStateDictionaryHelper(
```

Comment:
maybe just `constructAssetPackStateDictionary`? i.e. drop the `Helper`?

Reply:
Yeah that sounds good. Addressed.